t/plugin/skywalking-logger.t

```diff
@@ -0,0 +1,285 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+log_level('debug');
+repeat_each(1);
+no_long_string();
+no_root_location();
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    my $http_config = $block->http_config // <<_EOC_;
+
+    server {
+        listen 1986;
+        server_tokens off;
+
+        location /v3/logs {
+            content_by_lua_block {
+                local core = require("apisix.core")
+
+                core.log.warn(core.json.encode(core.request.get_body(), true))
+            }
+        }
+    }
+_EOC_
+
+    $block->set_value("http_config", $http_config);
```

Comment:
New tests should contain a common section, like https://github.com/apache/apisix/blob/f06f6cc0e8e41875bf105cf2c18457339002df53/t/plugin/limit-count2.t#L38

There is no need to repeat them.

Reply:
updated